master/buildbot/data/forceschedulers.py

```diff
@@ -24,6 +24,7 @@
 def forceScheduler2Data(sched):
     ret = dict(all_fields=[],
                name=unicode(sched.name),
+               button_name=unicode(sched.buttonName),
```

Comment:
it should default to the name for backward compatibility

and be documented (and this will probably break the tests)